### PR TITLE
fix(FEC-13209): [Audio Player] adjust full size player UI to entries of audio media type

### PR DESCRIPTION
**Pull request by SivanA-Kaltura**
Use ExpandableText component (which is also used in ui)

Related PRs:
https://github.com/kaltura/kaltura-player-js/pull/646
https://github.com/kaltura/playkit-js-ui/pull/783

Resolves FEC-13209

### CODE REVIEW

File: src/components/navigation/navigation-item/NavigationItem.tsx [hunk 1]
```diff
@@ -73,21 +73,14 @@ export class NavigationItem extends Component<NavigationItemProps, NavigationIte
   componentDidUpdate(previousProps: Readonly<NavigationItemProps>, nextState: Readonly<NavigationItemState>) {
     this._getSelected();
     this.matchHeight();
-    if (nextState.imageLoaded !== this.state.imageLoaded) {
```
Comment (semarche-kaltura) [inline src/components/navigation/navigation-item/NavigationItem.tsx:76]: don't we need image load handling anymore?

File: src/components/navigation/navigation-item/NavigationItem.tsx [hunk 2]
```diff
@@ -161,37 +137,6 @@ export class NavigationItem extends Component<NavigationItemProps, NavigationIte
...
-          className={styles.showMoreButton}
-          ref={node => {
-            if (expandText) {
-              this._showLessButtonRef = node;
-            } else {
-              this._showMoreButtonRef = node;
-            }
-          }}
-          aria-label={`${expandText ? this.props.readLessLabel : this.props.readMoreLabel} ${title}`}>
-          {expandText ? this.props.readLessTranslate : this.props.readMoreTranslate}
```
Comment (semarche-kaltura) [inline src/components/navigation/navigation-item/NavigationItem.tsx:182]: readLessTranslate,  readLessLabel, readMoreTranslate and readMoreLabel moved to `ExpandableText` component, can we remove translates from Navigation plugin

**Review by semarche-kaltura — CHANGES_REQUESTED** [review]

File: src/components/navigation/navigation-item/NavigationItem.tsx (hunk 1 above)
Comment (SivanA-Kaltura) [reply src/components/navigation/navigation-item/NavigationItem.tsx:76]: I think that titleTrimmed was only used to determine whether to show the show more text ? This is now handled by expandable-text

File: src/components/navigation/navigation-item/NavigationItem.tsx (hunk 2 above)
Comment (SivanA-Kaltura) [reply src/components/navigation/navigation-item/NavigationItem.tsx:182]: Oh, we already have translations ! I'll add them to playkit-ui then

Comment (SivanA-Kaltura) [reply src/components/navigation/navigation-item/NavigationItem.tsx:182]: When was show_more_of used vs show_more ?

Comment (semarche-kaltura) [reply src/components/navigation/navigation-item/NavigationItem.tsx:182]: according to code:
`show_more_of` uses in aria-label
`show_more` displays as button text

**Review by semarche-kaltura — APPROVED** [review]

**Pull request merged**